scripts/appleseedMaya/AETemplates/__init__.py

```diff
@@ -44,9 +44,13 @@ def __buildVisibilitySection(self):
         self.addControl('asVisibilityCamera', label='Camera')
         self.addControl('asVisibilityLight', label='Light')
         self.addControl('asVisibilityShadow', label='Shadow')
+        self.addControl('asVisibilityTransparency', label='Transparency')
```

Comment:
Is it possible to remove this commit?. Most if not all of the missing visibility flags are internal to appleseed and should work automatically. That is the case for NPR, Probe, Transparency and Subsurface I think.
The specular flag is not useful because there are no specular materials available in OSL and by extension in appleseed.maya.

Reply:
Done, thanks for the clarification.